src/content/guides/loading-and-sending-assets.mdx

```diff
@@ -18,8 +18,6 @@ If your asset type is not included on the list, contact integrations@centrapay.c
 
 You can send Assets such as Tokens and Giftcards by calling our [Asset Transfers](https://docs.centrapay.com/api/asset-transfers#asset-transfer-create) endpoint. You will need to have the recipient’s phone number for `recipientAlias` to identify the reciever.
 
-Optionally the `senderAlias` field can be used if you’re loading an asset on behalf of someone else. If the `recipientAlias` doesn’t end up creating a Centrapay account, then it will be sent to this number.
```

Comment:
**Suggestion:** We might still want to describe what happens here is the recipientAlias doesn't have an account, i.e. the asset will be returned to owner?

Reply:
Yep, done!